### PR TITLE
Update index.js in the View Status Component

### PR DESCRIPTION
**Pull request by deekshu25**
Addding a heading element to the ViewStatus Component

### CODE REVIEW

File: nitconf-frontend/src/components/ViewStatus/index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import './index.css';
+
+const ViewStatus = (props) => {
+  const {details} = props;
+  return (
+    <h1>Hello</h1>
+  );
```
Comment (hemanth-0007) [inline nitconf-frontend/src/components/ViewStatus/index.js:7]: Add some container and a table to show the status of the abstarct

File: nitconf-frontend/src/components/ViewStatus/index.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import './index.css';
+
+const ViewStatus = (props) => {
+  const {details} = props;
+  return (
+    <h1>Hello</h1>
```
Comment (hemanth-0007) [inline nitconf-frontend/src/components/ViewStatus/index.js:6]: Add some div container and table to view the required status of the abstract

**Review by hemanth-0007 — APPROVED** [review]
This is fine

**Review by Mohan-227 — APPROVED** [review]
no errors

**Pull request closed without merging**